installer/install-playos/install-playos.py

```diff
@@ -40,9 +41,15 @@ def find_device(device_path):
     boot_device = get_blockdevice("/iso")
     if boot_device is None:
         print("Could not identify installer medium. Considering all disks as installation targets.")
-        available_devices = all_devices
-    else:
-        available_devices = [device for device in all_devices if not device.path.startswith(boot_device)]
+
+    def device_filter(dev):
+        is_boot_device = (boot_device) and (dev.path.startswith(boot_device))
```

Comment:
Maybe by being a bit more explicit it's clearer that all devices pass this if the boot device could not be identified:

```suggestion
        is_boot_device = (boot_device is not None) and (dev.path.startswith(boot_device))
```